net-analyzer/nmap/metadata.xml

```diff
@@ -18,6 +18,7 @@
 		<flag name="ncat">Install the ncat utility</flag>
 		<flag name="nping">Install the nping utility</flag>
 		<flag name="nse">Include support for the Nmap Scripting Engine (NSE)</flag>
+		<flag name="symlink">Install symlink to nc</flag>
```

Comment:
I'd do it without an additional flag, just symlink once ncat is toggled on, otherwise looks like a mess

Reply:
Ok done. Thanks :)

Reply:
Wouldn't people want to use `ncat` while `nc` comes from traditional/openbsd netcat? Isn't it a conflict between both packages as soon as `ncat` is enabled?